sym_iwram.txt

```diff
@@ -73,10 +73,10 @@ build/sa2/src/core.o(.data);
 . = 0x000055E0; gNewInputCounters = .;
 . = 0x00005660; gWater = .;
 . = 0x000056A4; gUnknown_030056A4 = .;
-. = 0x000056B0; gUnknown_030056B0 = .;
-. = 0x000057B0; gUnknown_030057B0 = .;
-. = 0x00005830; gUnknown_03005830 = .;
-. = 0x00005834; gUnknown_03005834 = .;
+. = 0x000056B0; sPlayerStateBuffer = .;
```

Comment:
Consider using the `boost_mode_particles` file to generate these

Reply:
Done!